packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistory.tsx

```diff
@@ -15,8 +14,8 @@ import {
 import { MergedTransaction } from '../../state/app/state'
 import { TabButton } from '../common/Tab'
 
-const roundedTabClasses =
-  'roundedTab ui-not-selected:arb-hover relative flex flex-row flex-nowrap items-center gap-0.5 md:gap-2 rounded-tl-lg rounded-tr-lg px-2 md:px-4 py-2 text-base ui-selected:bg-white ui-not-selected:text-white justify-center md:justify-start grow md:grow-0'
+const tabClasses =
+  'text-white px-3 mr-2 ui-selected:border-b-2 ui-selected:border-white ui-not-selected:font-thin'
```

Comment:
<img width="1037" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/deb34d99-c587-4337-aec3-af5400056d8b">
according to Figma, the font weight is still 400 for not selected, but the text colour is different

Reply:
updated